data/json/items/gun/38.json

```diff
@@ -100,8 +108,16 @@
     "copy-from": "pistol_revolver",
     "looks_like": "sw_619",
     "type": "GUN",
-    "name": { "str_sp": "Ruger LCR .38" },
+    "name": { "str_sp": "conceal-carry six-shooter" },
     "description": "A compact, double-action-only revolver designed for easy concealment, with a stainless steel cylinder and aluminum frame.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "ruger_lcr_38",
+        "name": { "str": "Ruger LCR" },
+        "description": "The Ruger LCR (Lightweight Compact Revolver) is a modern revolver, starting production in just 2010.  It's a lightweight, aluminum-framed revolver, designed for conceal carry with it's double-action-only trigger, lightweight and compact size.  Due to the double-action trigger, it is more possible to carry it in your pocket instead of a holster due to the heavy trigger, but is still ill-advised."
```

Comment:
```suggestion
        "description": "The Ruger LCR (Lightweight Compact Revolver) is a modern revolver, starting production in just 2010.  It's a lightweight, aluminum-framed revolver, designed for conceal carry with its double-action-only trigger, lightweight and compact size.  Due to the double-action trigger, it is more possible to carry it in your pocket instead of a holster due to the heavy trigger, but still ill-advised."
```